templates/categories/awards.eta

```diff
@@ -1,8 +1,6 @@
-<section id="awards" class="awards">
+<section class="awards section" id="awards">
```

Comment:
Why you need a `.section` class ? The section tag should be enough

Reply:
because I have added some CSS to it